diff --git a/dist/shared/unstorage.zVDD2mZo.mjs b/dist/shared/unstorage.zVDD2mZo.mjs
index 3fc81092d109eef35f397e0dad492565e94c0b7a..9b254784a382a0c310ca38ce02cb4d29a2a94b24 100644
--- a/dist/shared/unstorage.zVDD2mZo.mjs
+++ b/dist/shared/unstorage.zVDD2mZo.mjs
@@ -128,7 +128,7 @@ function joinKeys(...keys) {
 }
 function normalizeBaseKey(base) {
   base = normalizeKey(base);
-  return base ? base + ":" : "";
+  return base;
 }
 function filterKeyByDepth(key, depth) {
   if (depth === void 0) {
